heat-stack/model/HeatLoadAnalysis.tsx

```diff
@@ -0,0 +1,12 @@
+export interface HeatLoadAnalysis {
+	rules_engine_version: string
+	estimatedBalancePoint: number
+	otherFuelUsage: number
+	averageIndoorTemperature: number
+	differenceBetweenBAndTBP: number
```

Comment:
`suggestion`: I think the first `BA` acronym here isn't quite right and should be `differenceBetweenTiAndTbp`

Reply:
ok